pkg/depgraph/callback.go

```diff
@@ -160,5 +160,25 @@ func prepareLegacyFlags(cfg configuration.Configuration, logger *log.Logger) { /
 		logger.Println("Init script:", initScript)
 	}
 
+	if cfg.GetBool(FlagNPMStrictOutOfSync) {
+		cmdArgs = append(cmdArgs, "--strict-out-of-sync")
+		logger.Println("Prevent testing out-of-sync lockfiles.\nIf there are out-of-sync lockfiles in the project, the test command fails when --strict-out-of-sync=true.")
```

Comment:
ah, i missed this earlier. these log messages are only for --debug output. so at the most they should only log the name and value of the arg.

this help text should rather go with the pflags in flags.go.